extensions/2.0/Vendor/EXT_structural_metadata/README.md

```diff
@@ -39,6 +39,7 @@ Written against the glTF 2.0 specification.
   - [Property Attributes](#property-attributes)
   - [Property Textures](#property-textures)
 - [Binary Data Storage](#binary-data-storage)
+- [Assigning Metadata](#assigning-metadata)
```

Comment:
It feels more appropriate for the `Assigning Metadata` section to go before `Binary Data Storage`